sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CanonicalizeSuite.scala

```diff
@@ -50,4 +52,30 @@ class CanonicalizeSuite extends SparkFunSuite {
     assert(range.where(arrays1).sameResult(range.where(arrays2)))
     assert(!range.where(arrays1).sameResult(range.where(arrays3)))
   }
+
+  test("Canonicalized result is not case-insensitive") {
```

Comment:
let's move it to `SameResultSuite`, also let's pick a simpler test, like using a `Project` with one columns instead of `Aggregate`.

Reply:
Ok,modified,thanks.